### PR TITLE
installed search/solr modules and file fixes

### PR DESCRIPTION
**Pull request by KMDOFNI**
installed search/solr modules and file fixes

### CODE REVIEW

File: project/config/src/config/core.extension.yml [hunk 1]
```diff
@@ -39,6 +38,8 @@ module:
   entity_reference_revisions: 0
   facets: 0
   facets_pretty_paths: 0
+  facets_summary: 0
+  fastly: 0
```
Comment (soda736) [inline project/config/src/config/core.extension.yml:42]: We should only install fastly on production.

File: project/config/src/config/core.extension.yml [hunk 2]
```diff
@@ -119,14 +121,14 @@ module:
   twig_tweak: 0
   ultimate_cron: 0
   unity_common: 0
-  unity_file_migrations: 0
   unity_frontpage: 0
   unity_image_styles: 0
   unity_internal_link_checker: 0
   unity_map_embed: 0
+  unity_search_pages: 0
+  update: 0
```
Comment (soda736) [inline project/config/src/config/core.extension.yml:129]: Update should not be installed.

**Review by soda736 — CHANGES_REQUESTED** [review]
Just the 2 modules to uninstall and any config related to them.

File: project/config/src/config/core.extension.yml (hunk 1 above)
Comment (KMDOFNI) [reply project/config/src/config/core.extension.yml]: Thanks , I have now updated this

File: project/config/src/config/core.extension.yml (hunk 2 above)
Comment (KMDOFNI) [reply project/config/src/config/core.extension.yml]: Thanks , I have now updated this

**Review by soda736 — APPROVED** [review]

**Pull request merged**